lens-macro/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+use proc_macro_hack::proc_macro_hack;
```

Comment:
I would still include at least a module-level docstring here, and likely also a copyright notice, because I'm anal-retentive like that.

Reply:
now that this is gone do I still need a dosctring somewhere? not sure where to put it